shimmingtoolbox/cli/mask.py

```diff
@@ -0,0 +1,153 @@
+#!/usr/bin/env python3
+
+import click
+import nibabel as nib
+import numpy as np
+import os
+
+import shimmingtoolbox.masking.threshold
+from shimmingtoolbox.masking.shapes import shape_square
+from shimmingtoolbox.masking.shapes import shape_cube
+
+CONTEXT_SETTINGS = dict(help_option_names=['-h', '--help'])
+
+
+@click.group(context_settings=CONTEXT_SETTINGS, help=f"Create a mask based on a specified shape (cube, square, etc.) "
+                                                     f"or based on the thresholding of an input image.")
+def mask_cli():
+    pass
+
+
+@mask_cli.command(context_settings=CONTEXT_SETTINGS,
+              help=f"Create a cube mask from the input file. Return the filename for the output mask.")
+@click.option('-input', 'fname_input', type=click.Path(), required=True, help="Input path of the nifti file to mask")
+@click.option('-output', type=click.Path(), default=os.curdir, help="Output folder for mask in nifti file")
+@click.option("-size", required=True, type=int, help="Length of the side of the cube along first, second and third "
+                                                     "dimension (in pixels)")
+@click.option("-center", nargs=3, type=int, default=(None, None, None), help="Center of the cube along first, second "
```

Comment:
@jcohenadad I wonder if the triple-quoted docstrings for click functions become redundant now that we use `sphinx_click` extension. can we forego those docstrings for any click functions and document them through these decorators instead?